chall04/spentti.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+print('input:')
+n = int(input())
+grid = []
+
+for row in range(n):
+	grid.append(list(input()))
```

Comment:
can be reduced  the way:

```python
grid = [list(input()) for _ in range(n)]
```

Plus, you are not using "row" so you should move it as a blank identifier `_`.